### PR TITLE
new policy tags

### PR DESCRIPTION
**Pull request by Aruldharanikumar**
This PR handles the addition tags to taxonomy

### CODE REVIEW

File: prod/schemas/policy_tags/sensitive_fields_updated_new.json
Comment (duygugencc) [inline prod/schemas/policy_tags/sensitive_fields_updated_new.json:1]: Regarding those changes, I run the test script to find any edge cases with hashed columns & exported results to this sheet below:
Seems like, we have to handle those either as a edge case or change the masking rule
[Sheet link](https://docs.google.com/spreadsheets/d/1QaYHYU1rgsjKP6K9c_0V3Ffdmo_vEgZW5rx_jV0Fzfk/edit?gid=1919412405#gid=1919412405)

File: prod/schemas/policy_tags/sensitive_fields_updated_new.json [hunk 1]
```diff
@@ -1000,7 +1020,65 @@
         "confidential": {}
     },
     "low_sensitivity_tags": {
-        "PII": {},
+        "PII": {
```
Comment (duygugencc) [inline prod/schemas/policy_tags/sensitive_fields_updated_new.json:1023]: Since we are introducing new PII fields, we need to verify whether they apply to the rest of the tables. If a match is found in legacy stack tables but the field is not classified as PII, we must account for this in the encryption template. Have you validated these cases? If not, let's ensure they are considered.

File: prod/schemas/policy_tags/sensitive_fields_updated_new.json
Comment (Aruldharanikumar) [reply prod/schemas/policy_tags/sensitive_fields_updated_new.json:1]: Great! I have changed the rules for 2 fields and other one is not part of this change. Its already handled

File: prod/schemas/policy_tags/sensitive_fields_updated_new.json (hunk 1 above)
Comment (Aruldharanikumar) [reply prod/schemas/policy_tags/sensitive_fields_updated_new.json:1023]: These changes contain fields which are only part of cdc, when we update the authviews of legacy stack we can add the new parents. It should be fine.

**Review by duygugencc — APPROVED** [review]
LGTM!
